quiz.js

```diff
@@ -0,0 +1,104 @@
+
+
+
+
+
+
```

Comment:
Generally huge hunks of comments like this is better suited to be in a README than inline in your code.  Not a huge issue that needs fixing for these quizzes/exercises, just a note for future work once you start doing your own thing that needs explaining.